base/applications/mspaint/winproc.cpp

```diff
@@ -157,6 +157,56 @@ void CMainWindow::InsertSelectionFromHBITMAP(HBITMAP bitmap, HWND window)
     ForceRefreshSelectionContents();
 }
 
+LRESULT CMainWindow::OnMouseWheel(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
```

Comment:
How does the mouse scrolling (not zoom) behave, if all the conditions happen:
1. The system returns some usual value for `SPI_GETWHEELSCROLLCHARS` (not too much),
2. But the monitor where paint is being tested is using a super-large resolution,
3. Paint has a very large picture opened that needs scrolling.
4. Zoom is at 100%.

Question: When scrolling, is it going to look like the image is scrolling "slowly" ? Or is it going to be proportional to the screen's (not the paint window) size? And of course it also needs to be "inversely" proportional to the current zoom level (if zoomed 200%, then one needs a scrolling of half the amount of what it was when the zoom was 100%).